packages/breadcrumbs/src/elements/Breadcrumb.tsx

```diff
@@ -27,7 +26,12 @@ export const Breadcrumb = React.forwardRef<HTMLElement, HTMLAttributes<HTMLEleme
         );
       }
 
-      return <StyledBreadcrumbItem>{child}</StyledBreadcrumbItem>;
+      return (
+        <StyledBreadcrumbItem>
+          {child}
+          <StyledChevronIcon />
```

Comment:
I'm not convinced this is an effective component layout. By making this icon a child of the item, you're failing to leverage the outer flexbox container which makes the `StyledBreadcrumbItem` need to function as both a flexbox container and flexbox item.

I'd recommend refactoring so that the `StyledChevronIcon` is a specific type of `StyledBreadcrumbItem` and is positioned as a sibling (aka direct child of `StyledBreadcrumb`).

Reply:
> I'd recommend refactoring so that the StyledChevronIcon is a specific type of StyledBreadcrumbItem

Thanks for pointing this out 👍. This was one of the alternatives I considered as a result of exploring breadcrumb components from other open source projects. It did not cross my mind to leverage the outer flexbox container, but that makes a lot of sense to do so. 

I will restructure the code so that the chevron is a direct sibling of the`StyledBreadcrumb`. That way, the UI can leverage the outer flexbox container.